py/server/deephaven/table_factory.py

```diff
@@ -65,11 +72,19 @@ def time_table(period: Union[str, int], start_time: str = None, blink_table: boo
     """
     try:
         builder = _JTableTools.timeTableBuilder()
+
+        if not isinstance(period, str) and not isinstance(period, int):
+            period = str(time.to_j_duration(period))
```

Comment:
The builder can accept a java Duration; it seems odd to wash that through a string, what am I missing?